qtpy/QtCore.py

```diff
@@ -58,6 +67,11 @@
     import PySide6.QtCore
     __version__ = PySide6.QtCore.__version__
 
+    # Missing QtGui utility functions on Qt
+    if getattr(Qt, 'mightBeRichText', None) is None:
+        from PySide6.QtGui import Qt as guiQt
```

Comment:
I think `guiQt` should be removed after the assignment in the line just before this one.